tracer/test/Datadog.Trace.SourceGenerators.Tests/InstrumentationDefinitionsGeneratorTests.cs

```diff
@@ -1698,7 +1698,7 @@ private static Payload GetInterfaceDefinitionsArray(InstrumentationCategory inst
         }
 
         [Fact]
-        public void CanGenerateIntegrationDefinitionForIast()
+        public void CanGenerateIntegrationDefinitionForIastWeakHash()
```

Comment:
Actually, I think we should remove this test + the weak cipher one. These tests are for testing the source generator itself, _not_ that the code generated by the integration is correct.

Reply:
Done!